scipy/stats/contingency.py

```diff
@@ -273,3 +275,124 @@ def chi2_contingency(observed, correction=True, lambda_=None):
                                    lambda_=lambda_)
 
     return chi2, p, dof, expected
+
+
+def association(n_obs, n_rows, n_cols, chi2_stat, stat="V"):
+    """Calculates degree of association between variables that are nominal or greater.
+
+    Allows for specification of one of four related methods, Tschuprow's T, Pearson's Contingency Coefficient,
+    Cramer's V and Phi.
+
+    Parameters
+    ----------
+    stat : {"V", "T", "C", "phi"} (default = "V")
+        The association test statistic.
+    n_obs : int or float
+        The total number of observations
+    n_rows : int or float
+        The total number of rows
+    n_cols : int or float
+        The total number of columns
+    chi2_stat : float
+        The chi squared statistic.
+    stat : {"V", "T", "C", "phi"} (default = "V")
+        The association test statistic.
+
+    Returns
+    -------
+    value : float
+        Value of the test statistic
+
+    References
+    ----------
+    .. [1] "Tschuprow's T",
+           https://en.wikipedia.org/wiki/Tschuprow's_T
+    .. [2] Bergsma, Wicher, "A bias-correction for Cramer's V and Tschuprow's T",
+           London School of Econ. and Pol. Sci., pp. 5-7.
+           http://stats.lse.ac.uk/bergsma/pdf/cramerV3.pdf
+    .. [3] Tschuprow, A. A. (1939) Principles of the Mathematical Theory of Correlation;
+           translated by M. Kantorowitsch. W. Hodge & Co.
+    .. [4] "Cramer's V", https://en.wikipedia.org/wiki/Cramer's_V
+    .. [5] "Nominal Association: Phi and Cramer's V",
+           http://www.people.vcu.edu/~pdattalo/702SuppRead/MeasAssoc/NominalAssoc.html
+    .. [6] Gingrich, Paul, "Association Between Variables", http://uregina.ca/~gingrich/ch11a.pdf
+
+
+    Examples
+    --------
+
+    2-way Example
+
+    >>> from scipy.stats.contingency import association, chi2_contingency
+    >>> obs = np.array([[100, 150], [203, 322], [42, 7], [32, 21]])
+    >>> num_obs = sum(obs.flatten())
+    >>> num_rows = obs.shape[0]
+    >>> num_cols = obs.shape[1]
+    >>> chi2 = tuple(chi2_contingency(obs))[0]
+
+    Pearson's contingency coefficient
```

Comment:
this needs a blank line between a line of text and the next `>>>` line to render correctly in the html docs. and same further down